controllers/os-ubuntu-alicloud/charts/os-ubuntu-alicloud/templates/os-coreos-alicloud-vpa.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
rename this file from `os-coreos-alicloud-vpa.yaml` to `os-ubuntu-alicloud-vpa.yaml`